### ISSUE TITLE
Vignette

### ISSUE THREAD

**Issue by jgeller112** [jaromilfrossard/permucoSlope#1]
Any plans to write a vignette?

**Comment by jaromilfrossard** [comment]
Yes, but it may take some time. A paper is planned to present this method. I think the vignette will be created in the same time.
